doc/source/tutorial-quickstart-pytorch.rst

```diff
@@ -4,270 +4,315 @@
 Quickstart PyTorch
 ==================
 
-.. meta::
-   :description: Check out this Federated Learning quickstart tutorial for using Flower with PyTorch to train a CNN model on MNIST.
+In this tutorial we will learn how to train a Convolutional Neural Network on CIFAR10 using Flower and PyTorch. It is recommended to create a virtual environment and run everything within a :doc:`virtualenv <contributor-how-to-set-up-a-virtual-env>`.
 
-..  youtube:: jOmmuzMIQ4c
-   :width: 100%
+Let's use `flwr new` to create a complete Flower+PyTorch project. It will generate all the files needed to run, by default with the Simulation Engine, a federation of 10 nodes using `FedAvg <https://flower.ai/docs/framework/ref-api/flwr.server.strategy.FedAvg.html#flwr.server.strategy.FedAvg>`_. The dataset will be partitioned using Flower Dataset's `IidPartitioner <https://flower.ai/docs/datasets/ref-api/flwr_datasets.partitioner.IidPartitioner.html#flwr_datasets.partitioner.IidPartitioner>`_.
 
-In this tutorial we will learn how to train a Convolutional Neural Network on CIFAR10 using Flower and PyTorch.
+Now that we have a rough idea of what this example is about, let's get started. We first need to create a PyTorch project. You can do this by running the command below. You will be prompted to give a name to your project as well as typing your developer name:
 
-First of all, it is recommended to create a virtual environment and run everything within a :doc:`virtualenv <contributor-how-to-set-up-a-virtual-env>`.
+.. code-block:: shell
+
+  $ flwr new --framework PyTorch
```

Comment:
Should we just ask people to use `flwr new` and follow the instructions (select PyTorch)? That would show them what other templates are available

Reply:
yes. i'll edit this